### ISSUE TITLE
Remove Scan for Games and Check for Emu Update from RetroAchievments titlebar

### ISSUE THREAD

**Issue by kdecks** [RetroAchievements/RAIntegration#178]
![image](https://user-images.githubusercontent.com/32706333/48311915-1fee4780-e564-11e8-9106-8d30d94b178e.png)

As far as I know, **scan for games** has never worked, and I'm not certain what the intention was for it. Perhaps to create a list of supported games in the overlay, or some other menu. Regardless it does not work, and so either it should be fixed or pruned.

**Check for Emulator** update is also unnecessary. The emulator will prompt you automatically at startup if an update is available.

**Comment by SyrianBallaS** [comment]
The main reason "Scan for games" doesn't work is because it only checks for Genesis ROMs (*.bin;*.gen). It should be relatively easy to fix. I was just waiting on the new rehash strategy to go live before doing this since hashes were always mismatched (rehash was different than Server MD5) anyway.

**Comment by kdecks** [comment]
@SyrianBallaS 

What happens when **Scan for games** works as intended?

As you see it what value might **Check for Emulator Update** have?

**Comment by SyrianBallaS** [comment]
I actually dont know is intended for it as there was no documentation on it and was there way before I started contributing. 

I can only guess it scans for roms and populates the list of it matches gamehashlibrary.txt in the racache folder. 
I was able to get yo scan the files but the md5 never matches.

I don't really think there's any value in checking for updates manually if it's already done.

I don't feel comfortable getting rid of all this.

**Comment by kdecks** [comment]
>I can only guess it scans for roms and populates the list of it matches gamehashlibrary.txt in the racache folder.
If one can get it to work then it offers utility.

>I don't really think there's any value in checking for updates manually if it's already done.

In my opinion if these features offers little or no utility then keeping them is a detriment; they are just visual garbage, or vestigial. I can't speak for @ScottFromDerby but I suspect he would agree.

**Comment by Jamiras** [comment]
I've removed the "Check for Emulator Updates" menu item in PR225. I've also commented out the "Scan for Games" menu item. The emulators are currently responsible for determining the proper way to hash their ROMs. Duplicating that logic in the toolkit seems undesirable (it's already duplicated in RALibRetro and RetroArch - but that's a topic for another discussion). I think there could be benefit in the feature if it worked, so I haven't removed all of the code associated to it yet, and as such, will leave this open even after 225 is merged.

**Comment by kdecks** [comment]
Here's what @ScottFromDerby said about it when I asked him about it near the beginning of this month:

> Hey,  yea it's been meaning to get back to you guys on that - I'd love for it to work, but I wouldn't be offended if you wanted to remove it. If you could disable or hide it we could hopefully get it working one day, but I can't see that happening soon unfortunately :(

That lines up with this action.
